bundler.d/aws.rb

```diff
@@ -0,0 +1,3 @@
+group :aws do
+  gem 'fog-aws'
```

Comment:
Isn't fog-aws added automatically to fog? (https://github.com/fog/fog/blob/master/fog.gemspec#L57)